.eslintrc

```
@@ -15,12 +15,12 @@
   },
   "rules": {
     "max-len": ["error", { "code": 140, "ignoreUrls": true }],
-    "import/prefer-default-export": false,
-    "@typescript-eslint/explicit-function-return-type": false,
+    "import/prefer-default-export": 0,
```

```suggestion
    "import/prefer-default-export": "off",
```

Let's use `"off"` instead of `0`, so it's easier to understand at a quick glance.